config/test/config.py

```diff
@@ -35,19 +35,16 @@
     ]
 )
 
-# Mandrill
-MANDRILL_API_KEY = env.get('MAIL_PASSWORD')
-MANDRILL_TRANSACTIONAL_TEMPLATE = 'notification-template'
-MANDRILL_TRANSACTIONAL_SUBACCOUNT = 'transactional'
-MANDRILL_ALERTS_TEMPLATE = 'notification-template'
-MANDRILL_ALERTS_SUBACCOUNT = 'alerts'
+# Sendgrid
+SENDGRID_API_KEY = env.get('SENDGRID_API_KEY')
+SENDGRID_TRANSACTIONAL_TEMPLATE_ID = '18988dde-425f-4abc-a62f-36f9221e3ff1'
 
 # Flask-Mail
-MAIL_SERVER = 'smtp.mandrillapp.com'
+MAIL_SERVER = 'smtp.sendgrid.com'
```

Comment:
For the test config, I'd ensure this is a fake value to avoid sending emails when testing by mistake. Ditto for the username and password.